cvrf2csaf/cvrf2csaf.py

```diff
@@ -7,8 +7,22 @@
 
 from .common.utils import get_config_from_file, store_json, critical_exit
 
-from .section_handlers.document_tracking import DocumentTracking
+from .section_handlers.document_acknowlegments import Acknowledgments
+from .section_handlers.document_aggregate_severity import AggregateSeverity
+from .section_handlers.document_category import DocumentCategory
+from .section_handlers.comment import Comment
```

Comment:
Where does this come from? I can't remember that there is a comment field in CVRF defined? Did I miss that?

Reply:
@tschmidtb51 You are right. There are about 58 matches in CVRF 1.2 spec for case insensitive comment but most of them are headings indicating non-normative comments and a handful in the boilerplate finally another handful motivating the Involvements container and items ... no XML element nor attribute with that name.

Reply:
`comment` has been observed in some of the examples. 

For further investigation, I wrote this small handler - But if we agree on dropping the field all in all, thats totally fine for me. Will remove this.